components/CardItem.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import logoImg from "@/public/img/svg/noImgLogo.svg";
+import starImg from "@/public/img/svg/star.svg";
+import kebabImg from "@/public/img/svg/kebab.svg";
+import mobileAddImg from "@/public/img/svg/mobileAdd.svg";
+import { getTimeDiff } from "../utils/postTime";
+import { isLocation } from "../utils/location";
+import { Links } from "../dataType/dataType";
+import Image from "next/image";
+import styles from "./cardItem.module.css";
+
+/* folder페이지는 props가 있고 shared페이지에는 없음 클릭함수타입지정 any */
+interface CardItemType {
+  item: Links;
+  prevKey?: number | null;
+  handleCebabClick?: any;
```

Comment:
함수도 이런 식으로 타입 지정해주시면 좋을 것 같습니다.
```suggestion
  handleCebabClick?: () => void
```